app/models/ems_event.rb

```diff
@@ -227,6 +227,7 @@ def get_target(target_type)
 
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
+    target_type = "middleware_server"   if event.event_type == "hawkular_event"
```

Comment:
Yes, I think we could need a more specific event_type for alert-related events.
(This task can be in a following PR as it needs some involvement in other areas).
Point is that with these use case we might have Hawkular Events that are visible for users and others that are part of the implementation.
For example, and undeployment event will be visible and it might be consumed by an end user.
But these hawkular events are internal as part of the implementation, so what it will be shown to the end user is the miq alert, not these events.

Middleware_server is the appropiate target_type for this end to end, but as commented, this will be extended as long as we include more alerting for other middleware models.